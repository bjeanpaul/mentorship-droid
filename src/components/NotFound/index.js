import React from 'react';
import { View, Text } from 'react-native';


/**
 TODO: Redirect after X minutes.
 TODO: Figure out if we can log this; and what information would be helpful
 to prevent this in future.
*/
const NotFound = () => (
  <View>
    <Text>
      You have stumbled into an unknown realm of the application.
      This is certainly a bug, and something that you should never have seen.
    </Text>
  </View>
);
export default NotFound;
