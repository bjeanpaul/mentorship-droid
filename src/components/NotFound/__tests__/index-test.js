import React from 'react';
import NotFound from 'src/components/NotFound';


describe('Not Found', () => {
  it('should render with a warning message', () => {
    expect(
      <NotFound />
    ).toMatchSnapshot();
  });

});
