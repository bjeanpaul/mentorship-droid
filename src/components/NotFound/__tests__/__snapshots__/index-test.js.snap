exports[`Not Found should render with a warning message 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    You have stumbled into an unknown realm of the application. This is certainly a bug, and something that you should never have seen.
  </Text>
</View>
`;
